docs/operator/troubleshooting.md

```diff
@@ -0,0 +1,107 @@
+
+## Starboard Pod Not Running
+
+The Starboard Operator will run a pod inside your cluster. If you have followed the installation guide, you will have installed the Operator to the `starboard-system`. If you have installed it to another namespace, make sure to adapt the commands below.
+
+Make sure that the pod is in the `Running` status:
+```
+kubectl get pods -n starboard-system
+```
+
+This is how it will look if it is running okay:
+
+```
+NAMESPACE            NAME                                         READY   STATUS    RESTARTS      AGE
+starboard-system     starboard-operator-6c9bd97d58-hsz4g          1/1     Running   5 (19m ago)   30h
+```
+
+If the pod is in `Failed`, `Pending`, or `Unknown` check the events and the logs of the pod.
+
+First, check the events, since they might be more descriptive of the problem. However, if the events do not give a clear reason why the pod cannot spin up, then you want to check the logs, which provide more detail.
+
+```
+kubectl describe pod <POD-NAME> -n starboard-system
+```
+
+To check the logs, use the following command:
+```
+kubectl logs deployment/starboard-operator -n starboard-system
+```
+
+If your pod is not running, try to look for errors as they can give an indication on the problem.
+
+If there are too many logs messages, try deleting the Starboard pod and observe its behaviour upon restarting. A new pod should spin up automatically after deleting the failed pod.
+
+## ImagePullBackOff or ErrImagePull
+
+Check the status of the Starboard Operator pod running inside of your Kubernetes cluster. If the Status is ImagePullBackOff or ErrImagePull, it means that the Operator either
+
+* tries to access the wrong image
+* cannot pull the image from the registry
+
+Make sure that you are providing the right resources upon installing the Starboard Operator.
+
+## CrashLoopBackOff
+
+If your pod is in `CrashLoopBackOff`, it is likely the case that the pod cannot be scheduled on the Kubernetes node that it is trying to schedule on.
+In this case, you want to investigate further whether there is an issue with the node. It could for instance be the case that the node does not have sufficient resources.
+
+## Reconsilation Error
+
+It could happen that the pod appears to be running normally but does not reconcile the resources inside of your Kubernetes cluster.
+
+Check the logs for reconcilation errors:
+```
+kubectl logs deployment/starboard-operator -n starboard-system
+```
+
+If this is the case, the Starboard Operator likley does not have the right configurations to access your resource. 
+
+## Operator does not create VulnerabilityReports
+
+VulnerabilityReports are owned and controlled by the immediate Kubernetes workload. Every VulnerabilityReport of a pod is thus, linked to a [replicaset.](./index.md) In case the Starboard Operator does not create a VulnerabilityReport for your workloads, it could be that it is not monitoring the namespace that your workloads are running on.
```

Comment:
```suggestion
VulnerabilityReports are owned and controlled by the immediate Kubernetes workload. Every VulnerabilityReport of a pod is thus, linked to a [ReplicaSet](./index.md). In case the Starboard Operator does not create a VulnerabilityReport for your workloads, it could be that it is not monitoring the namespace that your workloads are running on.
```